### DISCUSSION TITLE
Index to simulate join of a link table

### DISCUSSION THREAD

**Discussion by mcrio** [ravendb/ravendb#18101] [Q&A] [answered]
Hi, I have a problem figuring out this index:

Entities:

`Employee`: EmployeeId, StoreId
`Department`: DepartmentId, StoreId[]

Goal: Search employees and retrieve those that belong to given `DepartmentId`.
(you see how each emoployee has a storeId assigned, also each department has a list of storeIds it's managing)

I'm having a hard time figuring out an index that let's me do search on employees and filter by department as well.

**Comment by DejanMilicic** [comment, marked as answer]
Here is a working example https://github.com/DejanMilicic/RavenWorkshop/tree/main/Northwind/Northwind/Features/InnerJoin/DepartmentEmployees
